test/src/unit-ctx.cc

```diff
@@ -103,3 +105,31 @@ TEST_CASE("C++ API: Test context tags", "[cppapi][ctx-tags]") {
     }
   }
 }
+
+TEST_CASE("C++ API: Test REST version endpoint", "[cppapi][rest][version]") {
+  tiledb::test::VFSTestSetup vfs_test_setup;
+  tiledb::Config config;
+  std::string serialization_format = GENERATE("JSON", "CAPNP");
```

Comment:
Why have both sections and `GENERATE`?

Reply:
I see, I got misled by some bugs I had encountered in the past, but these were with sections, not dynamic sections.